easybuild/easyblocks/t/tensorflow.py

```diff
@@ -595,11 +595,12 @@ def configure_step(self):
             print_warning('Jemalloc is not supported in TensorFlow %s, the EC option with_jemalloc has no effect',
                           self.version)
         # Disable support of some features via config switch introduced in 1.12.1
-        if LooseVersion(self.version) >= LooseVersion('1.12.1'):
-            self.target_opts += ['--config=noaws', '--config=nogcp', '--config=nohdfs']
-            # Removed in 2.1
-            if LooseVersion(self.version) < LooseVersion('2.1'):
-                self.target_opts.append('--config=nokafka')
+        if LooseVersion(self.version) >= LooseVersion("1.12.1"):
+            self.target_opts += ["--config=nogcp"]
```

Comment:
If you touch this part of the code: Maybe we should either consistently use `+= [...]` or `.append` at least for single elements